paparazzi/src/test/java/app/cash/paparazzi/PaparazziTestRule.kt

```diff
@@ -0,0 +1,56 @@
+package app.cash.paparazzi
+
+import org.junit.rules.ExternalResource
+import org.junit.rules.TemporaryFolder
+import org.junit.runner.Description
+import org.junit.runners.model.MultipleFailureException
+import org.junit.runners.model.Statement
+
+class PaparazziTestRule : ExternalResource() {
```

Comment:
Could this be a `RuleChain` around Paparazzi instead? Doesn't necessarily need to manage Paparazzi too...

Reply:
No.  Instantiating Paparazzi will automatically instantiate the SnapshotHandler which has default values set to the system properties this rule stubs out.

Reply:
Got ya, thanks for explaining didn't catch that